extensions/amp-analytics/0.1/requests.js

```diff
@@ -162,7 +165,11 @@ export class RequestHandler {
 
       this.requestOriginPromise_ = this.variableService_
         // expand variables in request origin
-        .expandTemplate(this.requestOrigin_, requestOriginExpansionOpt)
+        .expandTemplate(
```

Comment:
ditto

Reply:
This one should definitely have the bindings, as it will include the `RESOURCE_TIMING` macro. Done.